misk/src/test/kotlin/misk/metrics/MetricsTest.kt

```diff
@@ -36,9 +36,9 @@ class MetricsTest {
 
         metrics.gauge("my_gauge", { n.get() })
         assertThat(metrics.gauges).containsKey("my_gauge")
-        assertThat(metrics.gauges["my_gauge"]!!.value).isEqualTo(0)
+        assertThat(metrics.gauges["my_gauge"]!!.value).isEqualTo(0L)
```

Comment:
This file just looks like a 4 -> 5 migration.

Reply:
See above